site/pages/about.md

```diff
@@ -0,0 +1,41 @@
+<!--
+.. title: About the NCAR Experimental Development Team
+.. slug: about
+.. date: 2019-10-21 15:46:50 UTC-06:00
+.. tags: 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+The NCAR Experimental Development (Xdev) Team is a small team of software engineers
+dedicated to the advancement and development of modern tools and techniques
+in scalable data analysis for geoscientists.  The Xdev Team was created as part
+of the [Pangeo](https://pangeo.io) collaboration.
+
+Currently, the Xdev Team consists of:
+
+- **Kevin Paul (Engineering Lead)**: Dr. Paul is a scientist and software engineer at NCAR with a
+  specialization in developing Python tools for data post-processing, most recently developing
+  tools to post-process the CESM contributions for CMIP6.
+- **Matthew Long (Science Lead)**: A scientist in the Climate & Global Dynamics laboratory
+  at NCAR with research specializations in modeling ocean ecosystems and biogeochemistry, 
+  interactions between ocean physics and biology, the global carbon cycle, ocean tracers and
+  stable isotope biogeochemistry impacts of climate change on ecosystems and biogeochemistry.
+- **Anderson Banihirwe**: Anderson is a software engineer at NCAR with a deep understanding
+  of the Python ecosystem, modern software best practices, and modern data analysis.
+- **Julia Kent**: Julia has a background in atmospheric and oceanic sciences and has recently
+  transitioned into software engineering with a specialty in working with scientists to develop
+  Python-based solutions to their workflows.
+- **Brian Bonnlander**: Brian is a computer scientist with a specialty in machine learning and
+  a strong motivation to provide high-level tooling to scientists that abstract way complex
+  computing issues.
+- **Micheal Levy**: Mike is a software engineer working on the Community Earth System Model
+  (CESM), where his main task is to maintain the Parallel Ocean Program (POP) repository.  Mike
+  has also helps with countless other small projects dealing with the CESM infrastructure.
+- **Mohammad Abouali**: Mohammad is a new software engineer at NCAR assisting with the 
+  [GeoCAT](https://geocat.ucar.edu/) team, assisting with the "Pivot to Python" NCL effort.
```

Comment:
Moe is a software engineer at NCAR on the [GeoCAT](https://geocat.ucar.edu/) team.